openshift/e2e-common.sh

```diff
@@ -100,7 +114,6 @@ function run_e2e_tests() {
 
   echo "Running e2e tests, directory ./test/e2e_channel/"
   go_test_e2e -timeout=100m -short ./test/e2e_channel/ \
-    -channels=messaging.knative.dev/v1beta1:KafkaChannel \
```

Comment:
Why is this not needed anymore?

Reply:
This means that this line is missing on v1.11 as I copied it from there

Reply:
I will revert